drivers/lora/sx1276.c

```diff
@@ -313,29 +313,19 @@ void SX1276SetRfTxPower(int8_t power)
 
 	if (power > 17) {
 		pa_dac = (pa_dac & RF_PADAC_20DBM_MASK) | RF_PADAC_20DBM_ON;
+		power -= 3;
```

Comment:
In case of 20DBM_ON there was `(power - 5)`, in the other case it was `(power - 2)`. So here comes the difference. But `power -= 3` can be confusing, I agree.